src/current/v23.1/migrate-from-postgres.md

```diff
@@ -6,254 +6,291 @@ keywords: copy
 
-If you only want to import one table from a database dump, see [Import a table from a full database dump](#import-a-table-from-a-full-database-dump) below.
+### Before you begin
 
-### Dump one table at a time
+The example uses a modified version of the PostgreSQL `french-towns-communes-francais` data set and demonstrates how to migrate the schema and data to a {{ site.data.products.serverless }} cluster. To follow along with these steps:
 
-To dump the `employees` table from a PostgreSQL database also named `employees`, run the [`pg_dump`][pgdump] command shown below. You can import this table using the instructions in [Import a table from a table dump](#import-a-table-from-a-table-dump) below.
+1. Download the `frenchtowns` data set:
 
-{% include_cached copy-clipboard.html %}
-~~~ shell
-$ pg_dump -t employees  employees > /tmp/employees.sql
-~~~
+       {% include_cached copy-clipboard.html %}
+       ~~~ shell
+       curl -O https://cockroachdb-migration-examples.s3.us-east-1.amazonaws.com/postgresql/frenchtowns/frenchtowns.sql
+       ~~~
 
-For this data set, the PostgreSQL dump file required the following edits, which have already been performed on the files used in the examples below.
+1. Create a `frenchtowns` database on your PostgreSQL instance:
 
-- The type of the `employees.gender` column in the `CREATE TABLE` statement had to be changed from `employees.employees_gender` to [`STRING`]({% link {{ page.version.version }}/string.md %}) since PostgreSQL represented the employee's gender using a [`CREATE TYPE`](https://www.postgresql.org/docs/10/static/sql-createtype.html) statement that is not supported by CockroachDB.
+       {% include_cached copy-clipboard.html %}
+       ~~~ shell
+       createdb frenchtowns
+       ~~~
 
-## Step 2. Host the files where the cluster can access them
+1. Load the `frenchtowns` data into PostgreSQL, specifying the path of the downloaded file:
 
-Each node in the CockroachDB cluster needs to have access to the files being imported. There are several ways for the cluster to access the data; for more information on the types of storage [`IMPORT`]({% link {{ page.version.version }}/import.md %}) can pull from, see the following:
+       {% include_cached copy-clipboard.html %}
+       ~~~ shell
+       psql frenchtowns -a -f frenchtowns.sql
+       ~~~
 
-- [Use Cloud Storage]({% link {{ page.version.version }}/use-cloud-storage.md %})
-- [Use a Local File Server]({% link {{ page.version.version }}/use-a-local-file-server.md %})
+1. Create a free [{{ site.data.products.db }} account](https://www.cockroachlabs.com/docs/cockroachcloud/create-an-account), which is used to access the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) and create the {{ site.data.products.serverless }} cluster.
```

Comment:
the product name isn't populated in the doc preview 🤔

Reply:
Hmm it's working on my local build, so I'll trust that this is ok! But good callout maybe for @nickvigilante ?

Reply:
@taroface Can you try rebasing this branch and force-pushing?

```
git checkout main
git pull
git rebase main
# fix merge conflicts
git rebase --continue
git push -fu origin HEAD
```

Reply:
Wait, I'm so silly. Gemma renamed the products here. I think this row is out of date.
```suggestion
1. Create a free [{{ site.data.products.cloud }} account](https://www.cockroachlabs.com/docs/cockroachcloud/create-an-account), which is used to access the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page) and create the {{ site.data.products.serverless }} cluster.
```